packages/quantic/cypress/e2e/default-2/result-list/folded-result-list/folded-result-list.cypress.ts

```diff
@@ -16,9 +22,10 @@ interface FoldedResultListOptions {
   parentField: string;
   childField: string;
   numberOfFoldedResults: number;
+  useCase: string;
 }
 
-describe('quantic-folded-result-list', () => {
+describe('quantic-folded-result-list', {browser: 'chrome'}, () => {
```

Comment:
In the Smart Snippet tests we added it because there was some issues related to the Salesforce Lightning Modal occurring when executing the tests in other browsers like "Electron" that comes with Cypress.
But none of this applied to the Folded Result List tests so we don't need it.